themes/default/data/registry/packages/nuage.yaml

```diff
@@ -0,0 +1,14 @@
+category: Cloud
+component: false
+description: ""
+featured: false
+logo_url: ""
+name: nuage
+native: true
+package_status: ga
+publisher: Nuage-Studio
+repo_url: https://github.com/nuage-studio/pulumi-nuage
+schema_file_path: schema.json
+title: nuage
+updated_on: 1675948534
```

Comment:
@burakgzt - there was an issue with the timestamp that was here. Previously it was  `-62135596800`. This time corresponds to the release of the package and should be the unix timestamp of that date. I updated it to when the release was made by looking at the v0.0.3 release that you published.

https://github.com/nuage-studio/pulumi-nuage/releases/tag/0.0.3